ui/DesignSystem/Component/SidebarLayout.svelte

```diff
@@ -8,8 +8,8 @@
 
   import Sidebar from "./Sidebar.svelte";
 
-  export let dataCy = null;
-  export let style = null;
+  export let dataCy = "";
+  export let style = "";
```

Comment:
I'm curious as to what this setup is for? Just for my self learning :)

Reply:
Figuring out the vibe for type inference v type declaration in `props`; still under construction 🚧